api/tests/unit/integrations/launch_darkly/test_services.py

```diff
@@ -60,20 +60,21 @@ def test_create_import_request__return_expected(
     [
         (
             HTTPError(response=MagicMock(status_code=503)),
-            "HTTP 503 when requesting LaunchDarkly",
+            "HTTPError 503 when requesting /expected_path",
         ),
-        (Timeout(), "Timeout when requesting LaunchDarkly"),
+        (Timeout(), "Timeout when requesting /expected_path"),
     ],
 )
 def test_process_import_request__api_error__expected_status(
     ld_client_mock: MagicMock,
     ld_client_class_mock: MagicMock,
     failing_ld_client_method_name: str,
-    exception: Type[Exception],
+    exception: Type[RequestException],
```

Comment:
I don't think this is `Type[RequestException]` is it? Isn't it just `RequestException` ? We're instantiating a `RequestException` object.

Reply:
Thanks! Fixed.